qiskit_superstaq/daily_integration_test.py

```diff
@@ -119,12 +119,13 @@ def test_qscout_compile_swap_mirror(provider: qss.SuperstaQProvider) -> None:
     out = provider.qscout_compile(qc, mirror_swaps=True)
     assert out.circuit == out_qc_swap
 
+    out = provider.qscout_compile(qc, mirror_swaps=False)
     op = qiskit.quantum_info.Operator(out.circuit)
     expected_op = qiskit.quantum_info.Operator(qc)
     assert op.equiv(expected_op)
 
     num_two_qubit_gates = 0
-    for op in qc:
+    for op in out.circuit:
         if len(op.qubits) > 1:
```

Comment:
should be something like
```suggestion
    for inst, _, _ in out.circuit:
        if inst.num_qubits == 2:
```